core/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/kubernetesModels.scala

```diff
@@ -438,7 +438,9 @@ final case class App(id: AppId,
                      descriptorPath: Option[Uri],
                      extraArgs: List[String],
                      sourceWorkspaceId: Option[WorkspaceId],
-                     numOfReplicas: Option[Int]
+                     numOfReplicas: Option[Int],
+                     autodeleteThresholdMin: Option[Int],
```

Comment:
```suggestion
                     autodeleteThreshold: Option[Int],
```
I believe (based on feedback to your questions in the doc) that we are going to drop the `Min` suffix altogether.

Reply:
Ah, I missed that suggestion, can you point me where it is because I want to follow up on that suggestion. The suggestion I found is to change to change `_min` to `Min`. 
Followup: If dropping min, how can I know the unit for both Leo client and developers?

Reply:
... I am looking for it. I know there was a whole conversation about it. ... 😬 (it's definitely not in the doc)

Reply:
[FOUND IT](https://broadinstitute.slack.com/archives/CA3NP1733/p1706719719692039?thread_ts=1706647144.938259&cid=CA3NP1733)! (whew - not crazy (yet))